### PR TITLE
Redo home page and add more fun facts

### PR DESCRIPTION
**Pull request by JosephBushagour**
Image of new page below:
Home_Page

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ function addRandomFunFact() {
     'I also have a hydroponic garden.', 
     'My favorite cereal is Cheerios.', 
     'When I was in third grade I refused to wear anything other than turtle-neck shirts.',
+    'I\'ve picked up the hobby of making my own hot sauces',
+    'I have an extensive collection of cheap pens.',
+    'I have acquired enough vintage electronic test equiment to start my own lab.',
```
Comment (SarahGB) [inline portfolio/src/main/webapp/script.js:26]: Nit: equipment

Comment (JosephBushagour) [reply portfolio/src/main/webapp/script.js]: Thanks, fix pushed!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -21,6 +21,11 @@ function addRandomFunFact() {
     'I also have a hydroponic garden.', 
     'My favorite cereal is Cheerios.', 
     'When I was in third grade I refused to wear anything other than turtle-neck shirts.',
+    'I\'ve picked up the hobby of making my own hot sauces.',
+    'I have an extensive collection of cheap pens.',
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/script.js:25]: What's the best cheap pen? I've always had a fondness for the Pilot G2

Comment (JosephBushagour) [reply portfolio/src/main/webapp/script.js:25]: The G2 is great! The 0.38mm version is my favorite cheap, widely available pen. The Zebra Sarasa is another really good choice, but it's less common to find in stores.

**Pull request merged**